<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMenu_1</class>
 <widget class="QDialog" name="UserMenu_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_data_godzina">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_wyloguj">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyloguj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Wstecz">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Wstecz</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_ZwrocWypozyczonySamochod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>751</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Zwróć wypożyczony samochód</string>
   </property>
   <widget class="QPushButton" name="pushButton_Zwroc">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zwróć</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Karta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Karta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_BLIK">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Blik">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>BLIK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Karta">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_UWAGA">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#ff0000;&quot;&gt;Kara za każdy dzień po terminie to 100 zł&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Kara">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>28</y>
      <width>127</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kara</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dni">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>127</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ile dni opóźnienia</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>194</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID Samochodu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_ID_Samochodu"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; font-style:italic;&quot;&gt;Aktualne Rezerwacje&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_AktualneRezerwacje">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>611</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
